src/RuleSet/Sets/SymfonySet.php

```diff
@@ -147,6 +147,10 @@ public function getRules(): array
                 ],
                 'sort_algorithm' => 'alpha',
             ],
+            'ordered_types' => [
```

Comment:
would be good to cover this set change in test.
ie tests/Fixtures/Integration/set/@Symfony.test

Reply:
It won't be compatible with the fact the test is running in PHP 7.4

Reply:
@keradus I don't see how to add this in Symfony.set since it must run in PHP 7.4 too.

But there is already test about this change since I needed to update the following code:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7356#issuecomment-1747299086